docs/ReleaseHistory_Rust.md

```diff
@@ -13,10 +13,14 @@ This document contains release notes pertaining to the Rust crates.
 - FNS => False negative reduction in static analysis.
 
 # UNRELEASED
+- FNS: Add detection of derived and hashed HIS v2 keys for `SEC101/200: Microsoft common annotated security key (HIS v2)`.
+
+# 1.5.4 - 11/19/2024
```

Comment:
This shipped without proper release notes. Copied below from the C# release notes.